v4_notes.md

```diff
@@ -0,0 +1,4 @@
+# Notes for Version 4
+
+`slice_coding_mode` was defined in a former version of the spec, it could be peek for a former version (<201-01-04)
```

Comment:
2018?
Also I suggest to link to the commit with the removal in the note, i.e. https://github.com/FFmpeg/FFV1/pull/93/commits/046648bbbf6c8ad3f277b49d26d18e5038f527ec.

Reply:
Fixed.